content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Allow_List.adoc

```diff
@@ -0,0 +1,114 @@
+== Allow list implementation guidance
+
+An allow list is a curated list of authenticators that should be permitted to register and authenticate to your application. Any authenticator that is not identifiable, or part of your curated list should be rejected. This scenario works well for applications that require a more secure and controlled environment, and where you want to ensure that your users are leveraging high assurance authenticators. The guidance given on this page will teach you how to implement an allow list in your application, that will limit registrations to a few select authenticators.
+
+This scenario works best when:
+
+* You are an organization that only uses a small set of specific authenticators
```

Comment:
Consider rewording:
* Your organization requires the use of a set of approved authenticators